crates/wasi-http/src/lib.rs

```diff
@@ -94,6 +94,7 @@
 //!     // component that we have. This will make per-request instantiation
 //!     // much quicker.
 //!     let mut linker = Linker::new(&engine);
+//!     wasmtime_wasi::p2::add_to_linker_async(&mut linker)?;
 //!     wasmtime_wasi_http::add_to_linker_async(&mut linker)?;
```

Comment:
I _think_ this needs to be
```suggestion
//!     wasmtime_wasi_http::add_only_http_to_linker_async(&mut linker)?;
```

otherwise it will end up shadowing e.g. `wasi:io`